examples/contracts/llm-erc20.py

```diff
@@ -0,0 +1,52 @@
+import json
+from genvm.base.equivalence_principle import EquivalencePrinciple
+
+
+class WizzardOfCoin:
+    prompt_part_1: str = """You keep track of transactions between users 
+    and their balance in coins. 
+    The current balance for all users in JSON format is:"""
+
+    prompt_part_2: str = "The transaction to compute is:"
+
+    prompt_part_3: str = """Given the current balance in JSON format and 
+    the transaction provided, please provide the updated balances as a 
+    JSON object.
+    For every transaction, validate that the user sending the Coins has 
+    enough balance. If any transaction is invalid it shouldn't be processed. 
+    Update the balances based on the valid transactions only.
+    It is mandatory that you respond with only the JSON balances object, nothing 
+    else whatsoever. Don't include any other words or characters, your 
+    output must be only the balances JSON object"""
```

Comment:
It's always better for the LLM performance to give an example of the output format

Also by the way you're describing this, it should like it should also say if the transfer is successful or not, and then revert the transaction based on that

Maybe something like

```
{
transaction_success: bool,
transaction_error: str, // Empty if transaction is successful
updated_balances: object<str, str> 
}

Reply:
Done!